.github/workflows/docs.yml

```diff
@@ -0,0 +1,40 @@
+
+name: Documentation
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    container:
+      image: 'archlinux:latest'
+
+    steps:
+      - name: Install dependencies
+        run: |
+          # assume yes and do not ask for confirmation when installing deps using pacman
+          yes | pacman -Syu base-devel sudo git wget curl python-sphinx rsync --noconfirm
+
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+
+      - name: Create builduser
+        run: |
+          # A user is required so that the docs are not built as root
+          useradd builduser -m # Create the builduser
+          passwd -d builduser # Delete the buildusers password
```

Comment:
why is this needed? sudo from root to a user won't ask for a password.

Reply:
We need `sudo` later in the script, in `makepkg` where it will install some deps from the `PKGBUILD`